src/components/Answer.js

```diff
@@ -0,0 +1,28 @@
+import React, {useState}from "react";
+
+
+const Answer = (answerObject) => {
+    const [selected, setSelected] = useState(false)
+    if (answerObject.content.img) {
+        return (
+            <img
```

Comment:
Misschien een idee om een Image component aan te maken, maar dit kan ook in een andere PR/issue aangekaart worden

Reply:
Ik heb ik de assets map een component Image aangemaakt. https://github.com/janesssa/prototype-social-warming/commit/68b8ebff9113189c2652b3236b67b34d60661bb8